src/docs/product/dashboards/widget-builder/index.mdx

```diff
@@ -0,0 +1,132 @@
+
+## Choose Your Columns
+
+The option to set columns is only available for [table visualizations](/product/dashboards/widget-builder/#table).
+
+By default, the first column is already configured for you, but you can update it. To add more columns, which stacks events, add [functions](/product/discover-queries/query-builder/#stacking-functions) (`f(x)`) that can take in parameters. [Field and tag columns](/product/sentry-basics/search/searchable-properties/#event-properties) will help you view more details about the events (for example, `title` to see the description of an issue or `users` to see how many users it's affecting).
+
+You can also add equations to your query using the table columns as equation variables, and the results will be displayed in the query results table. Learn more in [Adding Query Equations](/product/discover-queries/query-builder/query-equations/).
+
+Next to each column and equation, you have the option to set an alias that will replace the column name in the table header. 
+
+If two or more columns are added, you can change the order in which they're displayed in the table by clicking and dragging them:
+
+![Drag and drop column.](widget-builder-drag-and-drop-column.png)
+
+## Choose What to Plot in the Y-axis
+
+The option to set a y-axis is available on all visualizations except [table](/product/dashboards/widget-builder/#table). In the case of [time-series visualizations](/product/dashboards/widget-builder/#area-chart--bar-chart--line-chart), you'll be able to plot up to three values that can be functions or equations. If the units of your selections conflict, then the y-axis of the chart will be labeled using the units of the first selection.
+
+You can also choose to construct an equation by clicking “Add an Equation”. Doing this displays a text box where you can enter your equation. Functions that you've added previously in this step will appear for autocompletion, but you can add other functions as well.
+
+## Filter Your Results
+
+Use this field to filter your results by the searchable properties of your [dataset selection](/product/dashboards/widget-builder/#choose-your-dataset). For more information on which properties apply to each dataset, check out our [full Searchable Properties documentation](/product/sentry-basics/search/searchable-properties/#properties-table).
+
+On [time-series visualizations](/product/dashboards/widget-builder/#area-chart--bar-chart--line-chart) you can add up to three queries to filter your results, each with their own search conditions, to compare results. Each query will generate a new series in your chart for each y-axis. To make it easier to differentiate between them in the legend and tooltip, you can add a legend alias that will be added as a prefix to the labels, as in the example below:
+
+![Filter your results.](widget-builder-filter-your-results-with-three-queries.png)
```

Comment:
This is great! Let's crop this image a bit to make it easier to see the parts we're explaining; let's crop out the right sidebar entirely and the "Group your results" section since it's not relevant to the example.